tools/python_api/src_py/query_result.py

```diff
@@ -326,32 +326,28 @@ def encode_rel_id(rel: dict[str, Any]) -> tuple[int, int]:
                     continue
                 column_type, _ = properties_to_extract[i]
                 if column_type == Type.NODE.value:
-                    _id = row[i]["_id"]
-                    nodes[(_id["table"], _id["offset"])] = row[i]
-                    table_to_label_dict[_id["table"]] = row[i]["_label"]
+                    id = row[i]["_id"]
+                    nodes[id["table"], id["offset"]] = row[i]
+                    table_to_label_dict[id["table"]] = row[i]["_label"]
 
                 elif column_type == Type.REL.value:
-                    _src = row[i]["_src"]
-                    _dst = row[i]["_dst"]
                     rels[encode_rel_id(row[i])] = row[i]
 
                 elif column_type == Type.RECURSIVE_REL.value:
                     for node in row[i]["_nodes"]:
-                        _id = node["_id"]
-                        nodes[(_id["table"], _id["offset"])] = node
-                        table_to_label_dict[_id["table"]] = node["_label"]
+                        id = node["_id"]
+                        nodes[id["table"], id["offset"]] = node
+                        table_to_label_dict[id["table"]] = node["_label"]
                     for rel in row[i]["_rels"]:
                         for key in list(rel.keys()):
                             if rel[key] is None:
                                 del rel[key]
-                        _src = rel["_src"]
-                        _dst = rel["_dst"]
                         rels[encode_rel_id(rel)] = rel
 
         # Add nodes
         for node in nodes.values():
-            _id = node["_id"]
-            node_id = node["_label"] + "_" + str(_id["offset"])
+            id = node["_id"]
```

Comment:
Also here. In general, just double check to make sure there is not underscore removed from `_id`